crates/bitwarden-state/src/sdk_managed/configuration.rs

```diff
@@ -0,0 +1,17 @@
+#[derive(Debug)]
+/// Configuration for the database used by the SDK.
+pub enum DatabaseConfiguration {
+    /// SQLite configuration, used on native platforms
+    Sqlite {
+        /// The file path to the SQLite database. Databases for different users should be stored in
+        /// different files.
+        file_path: String,
```

Comment:
We could use `PathBuf` here, and just call `PathBuf::from(String)` from the Uniffi struct.

Non-blocking thought: SQLite tends to create a couple of files (the main db and the wal/shm files). It might be confusing if we receive a file path and also create a couple of extra files, should we take a directory instead? And then we can handle creating the files inside ourselves.

Reply:
Hmm yeah, in that case I can add `db_name` as an additional option so that we know what to name the files